components/templates/src/lib.rs

```diff
@@ -44,8 +44,11 @@ pub fn render_redirect_template(url: &str, tera: &Tera) -> Result<String> {
 }
 
 pub fn load_tera(path: &Path, config: &Config) -> Result<Tera> {
-    let tpl_glob =
-        format!("{}/{}", path.to_string_lossy().replace('\\', "/"), "templates/**/*.{*ml,md,txt}");
+    let tpl_glob = format!(
+        "{}/{}",
+        path.to_string_lossy().replace('\\', "/"),
+        "templates/**/*.{*ml,md,txt,json,ics}"
```

Comment:
So sorry I missed this for so long, I'm happy for this to resolve https://github.com/getzola/zola/issues/2772 and I assume it would resolve https://github.com/getzola/zola/issues/2768


My only question is why not just do the below?
```suggestion
        "templates/**/*"
```
Either way this PR fixes my use case so I'm happy for it to be merged as is and close https://github.com/getzola/zola/issues/2772, thank you!

Reply:
I had a previous PR with that, the reasons against it are [here](https://github.com/getzola/zola/pull/2853#issuecomment-2810397356).